package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "money-math",
-  "version": "2.4.4",
```

Comment:
According to SemVer convention, it should be a minor version bump ­— 2.5.0. I'll bump it myself though, when tagging with `npm version`